src/CSSBattle/3_PushButton/Tin.html

```diff
@@ -0,0 +1,59 @@
+<body>
+  <div class = rectangle>
```

Comment:
would be good to standardise the way you specify values for the class (i.e. class="rectangle" vs class = rectangle). putting quotation marks makes it more readable, more standardised and allows you to specify more than one class if necessary

Reply:
commmit 4ebbee1243e993add9f48d39646579cb86fc386b